homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -206,6 +172,32 @@ def __init__(
         else:
             self._masked_id = None
 
+        self._apply_event(event)
+
+    async def async_added_to_hass(self):
+        """Restore RFXtrx switch device state (ON/OFF)."""
+        await super().async_added_to_hass()
+
+        def _handle_event(event):
+            """Check if event applies to me and update."""
+            if event.device.id_string != self.event.device.id_string:
+                return
+
+            _LOGGER.debug(
+                "Binary sensor update (Device ID: %s Class: %s Sub: %s)",
+                slugify(event.device.id_string.lower()),
```

Comment:
Do we really need to use slugify in the log message? I'd expect the log to just log the event details as the come. If we want to log the device id that should be done with a cached attribute.

Reply:
Nope. I'll drop this in next rounds